### DISCUSSION TITLE
Script hangs at user input

### DISCUSSION THREAD

**Discussion by NickBarnes574** [jdjaxon/linux_cac#32] [Q&A]
Hello, I keep getting an error where the script hangs when asking for user input:

![image](https://github.com/user-attachments/assets/a9aded63-ff16-4854-a375-02b6b88c1a64)

I'm not able to provide any user input, nor am I able to `ctrl+C`, or `ctrl+D`out of the terminal. I have to force quit the terminal each time, and haven't been able to get any further.

Specs:
```
OS: Ubuntu 22.04
CPU: x86-64
```

Thanks for any help

**Comment by jdjaxon** [comment]
@NickBarnes574

How did you run the script?

I'm going to try to reproduce the error to try to determine if a recent merge caused this or if this is something new. Additionally, you should open this as an issue. This definitely seems like a bug. I'm not sure why the output for the prompt looks so wild. My apologies for the inconvenience.

**Reply by NickBarnes574** [reply]
Hey no problem! Appreciate the reply. I ran the script like so:
`sudo bash -c "$(wget https://raw.githubusercontent.com/jdjaxon/linux_cac/main/cac_setup.sh -O -)"`

**Reply by jdjaxon** [reply]
Okay. Thank you! I'll try to reproduce this when i have time, which will hopefully be tomorrow. Once I have something, I'll resume this conversation in issue #33.
